scripts/qa_crowbarsetup.sh

```diff
@@ -2924,6 +2924,7 @@ function oncontroller_testsetup()
 
     # do volume tests for non-docker scenario only
     if [ -z "$want_docker" ] ; then
+        ssh $ssh_target "modprobe acpiphp"
```

Comment:
no, volume attachment/detachment needs guest kernel coorporation, so it needs to be loaded in the guest kernel (there is a PCI hotplug protocol that the guest VM needs to respond to so that OpenStack can detach/attach volumes on the fly).